### PR TITLE
Add information about how to store and use keys in fastlane

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:warning:</td>
      <td data-sticky="true"><code>mkdocs.yml</code> was modified - make sure no modification was made by mistake</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: docs/best-practices/Keys.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+You can edit your `~/.bash_profile` to include something like
+
+```sh
+export SLACK_URL="https://hooks.slack.com/services/T03NA19Q5/..."
+export CRASHLYTICS_API_TOKEN="123abc"
+```
+
+If you use a different shell (e.g. `zshell`) you'll need to edit `~/.zshrc` instead.
+
+After editing the `bash_profile` you need to either re-open your terminal session or run `source ~/.bash_profile`.
```
Comment (mfurtak) [inline docs/best-practices/Keys.md]: quick fix file name to be `.bash_profile` at beginning of sentence

**Comment by mfurtak** [comment]
Great guide - I learned some things! 🙌

**Comment by mfurtak** [comment]
🚢  when ready

**Pull request merged**